mlir/lib/Target/LLVMIR/ModuleImport.cpp

```diff
@@ -473,9 +487,17 @@ ModuleImport::processAliasScopeMetadata(const llvm::MDNode *node) {
       StringAttr description = nullptr;
       if (!aliasScope.getName().empty())
         description = builder.getStringAttr(aliasScope.getName());
+      Attribute idattr;
+      if (verifySelfRef(scope))
+        idattr = DistinctAttr::create(builder.getUnitAttr());
+      else {
+        auto Name = cast<llvm::MDString>(scope->getOperand(0));
+        idattr = builder.getStringAttr(Name->getString());
+      }
```

Comment:
```suggestion
      Attribute idAttr;
      if (verifySelfRef(scope)) {
        idAttr = DistinctAttr::create(builder.getUnitAttr());
      } else {
        auto name = cast<llvm::MDString>(scope->getOperand(0));
        idAttr = builder.getStringAttr(name->getString());
      }
```
nit: same style nits as above